snap/hooks/configure

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Make sure we have lxd installed to use
+snap install lxd || true
```

Comment:
I actually decided to take another approach to this: https://github.com/conjure-up/conjure-up/blob/master/snap/hooks/configure#L3-L40

This allows us to utilize the deb installed LXD on Xenial servers and above and for desktops and Trusty will utilize the snap LXD. It's not ideal but at least you won't run into that issue of 2 LXD's trying to complete.